package.json

```diff
@@ -4,11 +4,12 @@
   "description": "Angular powered Bootstrap",
   "author": "https://github.com/ng-bootstrap/ng-bootstrap/graphs/contributors",
   "engines": {
-    "node": ">=10.9",
-    "yarn": ">=1.3.0 <2.0.0"
+    "node": ">=12.10",
+    "yarn": ">=1.15.2 <2.0.0"
   },
   "scripts": {
     "preinstall": "node misc/preinstall.js",
+    "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
```

Comment:
This `ngcc` script seems to be removed from the package file already, as creating a new or fresh Angular 9 project would not include this on its `postinstall` script. (Although this was on rc.8 version and not the rc.7 one). Cheers! 🎉

Reply:
Thanks for the tip @RinMinase. At the moment unit tests are failing if this is not done on `postinstall` for some reason. We'll have to investigate this and probably remove later